### ISSUE TITLE
List all available rooms 

### ISSUE THREAD

**Issue by iamshubhankarkhare** [iamshubhankarkhare/Chat-application#5]

**Comment by shobhit10058** [comment]
Hi, I think you want to say that all the available rooms should be shown when a user tries to login. I would like to work for this issue. I think we need to dynamically store the rooms in the backend.

**Comment by iamshubhankarkhare** [comment]
yes..there should be a button below login button which shows a list of all available rooms..we are already storing all the rooms ..you just have to list all the rooms
